WTWSMS/events/religious_events.txt

```diff
@@ -3763,11 +3809,16 @@ character_event = {
 			
 			# Pragmatism
 			modifier = {
-				factor = 3  # Powerful rulers want a modern religion
+				factor = 4  # Powerful rulers want a modern religion
 				tier = KING
 				realm_size = 12
 			}
 			modifier = {
+				factor = 6
+				tier = EMPEROR
+				realm_size = 24
+			}
+			modifier = {
```

Comment:
Twenty four is the realm size a King or Emperor needs for the modifier that makes him more likely to convert fire.